doc/ci_admin_guide/Windows_CI_logs_layout.md

```diff
@@ -0,0 +1,47 @@
+Windows CI logs layout
+======================
+
+This document describes a log structure generated by `ci-contrail-windows-production` job.
+One can access these logs by following a link `ci-contrail-windows-production` in the comment posted by `Contrail Windows CI` used, after a job has finished.
+
+## Log structure
+
+- `log.txt.gz` - full Jenkins job log
+- `unittests-logs` - logs for vRouter unit tests ran by Windows CI
+- `TestReports/CISelfcheck` - test report for Windows CI unit tests suite
+    - for information about more detailed logs please refer to [CI Selfcheck](#ciselfcheck)
+- `TestReports/WindowsCompute` - test report for Contrail Windows integration tests suite
+    - for information about more detailed logs please refer to [WindowsCompute](#windowscompute)
```

Comment:
Please check 'http://logs.opencontrail.org/winci/c9cbcbca291d4a2cad01ce89d51f122e/TestReports/SconsTestsLogs' and 'http://logs.opencontrail.org/winci/c9cbcbca291d4a2cad01ce89d51f122e/unittests-logs'.
Where is that discrepancy from?

Reply:
The change introducing `unittests-logs` is not yet merged. Sorry, should have mentioned that in the PR description.

Here's the PR: https://github.com/Juniper/contrail-windows-ci/pull/234